### PR TITLE
Change SMP limit in Memory Allocation Limits Table

### PR DESCRIPTION
**Pull request by nahceigna**
Changed maximum SMP to 2013GB in Memory Allocation Limits Table to avoid confusion.

Issue #1991

### CODE REVIEW

**Comment by jkwmoore** [comment]
Not sure we should imply that the very large memory nodes are the "regular" maximum RAM as this is a little misleading. (Given the present value is the normal node maximum RAM size.)

Queueing to large RAM nodes will take longer and I really do not want to encourage users to go ham on the RAM requests.

But I agree we should make a change here to indicate the highest limit. Not sure what form this should take.

**Comment by jkwmoore** [comment]
Hmm, this looks kinda whack in terms of layout:

image

Should we refactor the table to make the (Default/Max) more obvious as separate subcolumns under a common "Interactive Job" or "Batch Job" column.?

**Review by nikishe — APPROVED** [review]
Nice improvements on the table layout

File: referenceinfo/imports/scheduler/memory_allocation_limits_table_import.rst [hunk 1]
```diff
@@ -1,33 +1,31 @@
...
+  +------------------------------------------------------------------------------+------------------------------------------------+------------------------------------------------+
+  | **Very Large RAM Nodes**                                                     | 2014 MB                                        | N/A                                            |
+  +-----------------------------------+------------------------------------------+------------------------------------------------+------------------------------------------------+
+  | **Interactive Job**               | **Maximum Possible Request**             | 251 GB                                         | 192 GB                                         |
+  +-----------------------------------+------------------------------------------+------------------------------------------------+------------------------------------------------+
+  | **Batch Job (SMP)**               | **Maximum Regular Node Request**         | 2014 GB                                        | 192 GB                                         |
```
Comment (Carldkennedy) [inline referenceinfo/imports/scheduler/memory_allocation_limits_table_import.rst:20]: Stanage max and regular values need swapping

File: referenceinfo/imports/scheduler/memory_allocation_limits_table_import.rst [hunk 2]
```diff
@@ -1,33 +1,31 @@
...
+  +------------------------------------------------------------------------------+------------------------------------------------+------------------------------------------------+
+  | **Standard Nodes**                                                           | 251 MB                                         | 192 GB                                         |
+  +------------------------------------------------------------------------------+------------------------------------------------+------------------------------------------------+
+  | **Large RAM Nodes**                                                          | 1007 GB                                        | N/A                                            |
+  +------------------------------------------------------------------------------+------------------------------------------------+------------------------------------------------+
+  | **Very Large RAM Nodes**                                                     | 2014 MB                                        | N/A                                            |
```
Comment (Carldkennedy) [inline referenceinfo/imports/scheduler/memory_allocation_limits_table_import.rst:16]:
```suggestion
  | **Very Large RAM Nodes**                                                     | 2014 GB                                        | N/A                                            |
```

File: referenceinfo/imports/scheduler/memory_allocation_limits_table_import.rst [hunk 3]
```diff
@@ -1,33 +1,31 @@
...
+  |                                   | **Maximum Possible Request**             | 251 GB                                         | 192 GB                                         |
+  +-----------------------------------+------------------------------------------+------------------------------------------------+------------------------------------------------+
+  | **Batch Job (MPI)**               | **Maximum Possible Request**             | ~74404 GB                                      | 192 GB                                         |
+  +-----------------------------------+------------------------------------------+------------------------------------------------+------------------------------------------------+
+  | **Submission Argument**                                                      | --mem=<nn>                                     | --mem=<nn>                                     |
+  | **On a per node (job) basis**                                                |                                                |                                                |
```
Comment (Carldkennedy) [inline referenceinfo/imports/scheduler/memory_allocation_limits_table_import.rst:27]: No need to have this on two lines. Lowercase 'on'

File: referenceinfo/imports/scheduler/memory_allocation_limits_table_import.rst [hunk 4]
```diff
@@ -1,33 +1,30 @@
...
-     - 192 GB
-     - N/A
-     - N/A
-     - 2 GB / 192 GB
-     - 2 GB / 192 GB
-     - **Per node (job) basis** ``--mem=<nn>``
-
-
+.. table:: Memory Allocation Limits Table
+  :align: left
```
Comment (Carldkennedy) [inline referenceinfo/imports/scheduler/memory_allocation_limits_table_import.rst:3]:
```suggestion
```

Comment (Carldkennedy) [reply referenceinfo/imports/scheduler/memory_allocation_limits_table_import.rst:3]: Causes the following text to wrap. i.e alongside the table

**Review by Carldkennedy — APPROVED** [review]
One small change required otherwise LGTM. Nice work.

**Comment by jkwmoore** [comment]
I'm certainly guilty of not doing it quite often but can folks please squash merge where appropriate. Ta!

**Pull request merged**